modules/analysis-runner/src/cljdoc/analysis/deps.clj

```diff
@@ -103,15 +102,17 @@
 
 (defn resolved-and-cp
   "Build a classpath for the project specified by `pom-url`."
-  [pom-url repos]
+  [jar-url pom-url repos]
   {:pre [(string? pom-url)]}
```

Comment:
A similar assertion could be added for the `jar-url` I guess.

Reply:
done